CHANGES.rst

```diff
@@ -49,6 +49,9 @@ Cubeviz
 
 - Fixed validation message of moment number in moment map plugin. [#1536]
 
+- Fixed ``app._jdaviz_helper`` returning Specviz helper instead of Cubeviz helper when Specviz
+  helper is called. ``app._jdaviz_helper`` always returns Cubeviz helper [#1546]
```

Comment:
Hidden attribute should be hidden from user, so I propose a little more changes to the verbiage. 😬 

```suggestion
- Fixed ``viewer.jdaviz_helper`` returning Specviz helper instead of Cubeviz helper after Specviz
  helper is called via ``Cubeviz.specviz``. Now ``viewer.jdaviz_helper`` always returns the Cubeviz helper. [#1546]
```

Reply:
Ok Happy to adopt. I changed it to the hidden attribute because that was the root cause, but I can see your point